civic_to_bedpe.py

```diff
@@ -0,0 +1,126 @@
+
+
+'''
+Convert EricScript output to BEDPE format.
```

Comment:
`Convert CIViC database format to BEDPE format

Reply:
oops forgot to change the comments

Reply:
done